.github/workflows/icon-review.yml

```diff
@@ -54,19 +55,29 @@ jobs:
           echo image_url=$IMAGE_URL >> $GITHUB_ENV
         shell: bash
 
-      - name: Post review in PR ✍️
-        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+      - name: Find comment if exists 🕵️
+        uses: peter-evans/find-comment@3eae4d37986fb5a8592848f6a574fdf654e61f9e # v3.1.0
+        id: fc
         with:
-          script: |
-            const image_url = "${{ env.image_url }}";
-            console.log('URL: '+image_url);
-            const commentBody = '## Preview \n\nThank you for creating a pull request. This preview shows you how your changes will look on the different themes: \n\n ![Generated Preview](' + image_url + ') \n\n You can find more information how to contribute in the [contribution guidelines](https://github.com/material-extensions/vscode-material-icon-theme/blob/main/CONTRIBUTING.md).';
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: This preview shows you how your changes will look on the different themes
 
-            github.rest.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: commentBody
-            })
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Post or update comment in PR ✍️
+        uses: peter-evans/create-or-update-comment@71345be0265236311c031f5c7866368bd1eff043 # v4.0.0
+        with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            ## Preview
+
+            Thank you for creating a pull request. This preview shows you how your changes will look on the different themes:
+
+            <details ${{ env.svgFiles > 10 && 'open' || '' }}>
+              <summary>Generated Preview</summary>
```

Comment:
AAOOAOQOQOOA ITS NOT HALAL CODE!
PLIZ STOP 🛑 MAKING THIS CoDe

Better leave it to me.

Reply:
??

Reply:
Better now?

Reply:
@okineadev Please let us work together professionally and exchange ideas in a constructive and understandable way! Can you please tell us what you think could be improved? 
@lishaduck I really like your changes, let's try it out and see if it works 👍

Reply:
@lishaduck, Can I commit to your branch?

Because making suggestions is a bit tricky

Reply:
@lishaduck @okineadev Is the problem that `env.svgFiles > 10` might not work? I'm wondering if this can work, because `env.svgFiles` is just a string right? We should count the amount of files first and then put it into the condition, if I understood it correctly.

Reply:
I can grant you commit access in a few minutes, @okineadev.

Reply:
@PKief, GH should coerce it to a string. The docs also mentioned a JSON parser if you needed to be more explicit about it.
I might just make this edit an image and test it that way.

Reply:
> I can grant you commit access in a few minutes, @okineadev.

You should have commit access to my fork now. @PKief should have it automatically to branches I've PRed.